src/App.tsx

```diff
@@ -1,47 +1,110 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import './App.css';
-import Header from './components/Header';
+import axios from 'axios';
+import queryString from 'query-string';
+import { Innholdsformasjon } from './types';
 import CollapsedBox from './components/CollapsedBox';
-import { TitleLabel, AuthorLabel, ISBNLabel, ImageContainer } from './components/CustomElements';
-import pelsjegere from './resources/pelsjegere.jpg';
+import Header from './components/Header';
+import { AuthorLabel, ErrorTextField, ImageContainer, ISBNLabel, TitleLabel } from './components/CustomElements';
 
 const URL = window.location.href;
```

Comment:
Finnes egen react-hook for å lytte på location: "const location = useLocation()". Den må da i så fall plasseres inni react-komponenten